frontend/scripts/build-frontend-simple.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+if ! which purescript-derive-lenses >/dev/null; then
+    echo "Installing purescript-derive-lenses..."
+    rm -rf purescript-derive-lenses
+    git clone https://github.com/paf31/purescript-derive-lenses
+    cd purescript-derive-lenses
+    git checkout 02457e6 # TODO: I am not sure which version should we use? @jens?
```

Comment:
That's the correct commit of `purescript-derive-lenses` (see https://github.com/input-output-hk/cardano-sl-explorer/blob/cardano-sl-0.5-rc/frontend/purescript-derive-lenses.nix#L10)